### ISSUE TITLE
Can't add multiple tags at once to mock EC2 instance

### ISSUE THREAD

**Issue by dtenenba** [spulec/moto#754]
Hi,

According to the [AWS documentation](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/allocation-tag-restrictions.html), a resource can have a maximum of 50 tags.

However, in moto I cannot add more than one tag at once to a mock ec2 instance, but it works in 'real' ec2.:

```python
import moto
import boto3

# try the same on 'real' ec2 by commenting out the next 2 lines
mock = moto.mock_ec2()
mock.start()

client = boto3.client("ec2")
instance = client.run_instances(ImageId="ami-5ec1673e", MinCount=1, MaxCount=1)
instance_id = instance['Instances'][0]['InstanceId']

tags = []
for i in range(12):
    tags.append({"Key": "Name{}".format(i), "Value": "Value{}".format(i)})


client.create_tags(Resources=[instance_id], Tags=tags) # throws error below
mock.stop() # comment this out if you are trying this on 'real' ec2
```

Throws this error:

```
---------------------------------------------------------------------------
ClientError                               Traceback (most recent call last)
<ipython-input-96-e927403a88c3> in <module>()
----> 1 client.create_tags(Resources=[instance_id], Tags=tags)

/Users/dtenenba/envs/gemstrigger/lib/python2.7/site-packages/botocore/client.pyc in _api_call(self, *args, **kwargs)
    157                     "%s() only accepts keyword arguments." % py_operation_name)
    158             # The "self" in this scope is referring to the BaseClient.
--> 159             return self._make_api_call(operation_name, kwargs)
    160
    161         _api_call.__name__ = str(py_operation_name)

/Users/dtenenba/envs/gemstrigger/lib/python2.7/site-packages/botocore/client.pyc in _make_api_call(self, operation_name, api_params)
    492
    493         if http.status_code >= 300:
--> 494             raise ClientError(parsed_response, operation_name)
    495         else:
    496             return parsed_response

ClientError: An error occurred (TagLimitExceeded) when calling the CreateTags operation: The maximum number of Tags for a resource has been reached.
```

I can add the tags one at a time.

```python
for tag in tags:
    client.create_tags(Resources=[instance_id], Tags=[tag])
```

But that involves 12 API calls, and I shouldn't have to change my code.

**Comment by dtenenba** [comment]
Update: the actual issue seems to be that moto has a limit of 10 tags per resource. I can add ten tags (or fewer) to an instance, either all at once or one at a time. But no more than that. I'm submitting a PR to fix this.

**Comment by spulec** [comment]
Fixed with #755
